src/loki/search.cc

```diff
@@ -532,6 +535,13 @@ struct bin_handler_t {
       bool all_prefiltered = true;
       for (p_itr = begin; p_itr != end; ++p_itr, ++c_itr) {
         c_itr->sq_distance = std::numeric_limits<double>::max();
+        // TODO: for closings its possible that we could use the opposing edge if edge is filtered
```

Comment:
imho we should solve this now so we dont have to debug this later :smile: its hard to write a test for this becasue it depends on order of edges in the data but i think we can just fix the logic and allow the rest of the tests to pass as a good enough signal its not broken anything.

Reply:
ill take care of it tomorrow or so